website/versioned_docs/version-0.13.0/releases.mdx

```diff
@@ -10,6 +10,25 @@ import TierLabel from "./_components/TierLabel";
 :::note
 This page details the changes for Weave GitOps Enterprise and its associated components. For Weave GitOps OSS - please see the release notes on [GitHub](https://github.com/weaveworks/weave-gitops/releases).
 :::
+## v0.13.0
+2022-12-22
+### Highlights
+
+- Add Workspaces List and details in the UI
+- Bug fixes and improvements 
```

Comment:
I think highlights only mention things explicity, so I feel tha `bug fixes and improvement` doesnt really count as highlights and I've checked previous releases highlights and I see nothing similar there.